dashboard/test/ui/features/curriculum_platform/levelbuilder/lesson_edit_page.feature

```diff
@@ -56,7 +53,7 @@ Feature: Using the Lesson Edit Page
     # confirmed earlier that there were no Artist levels in the initial view.
     And I wait until element ".uitest-level-dialog-content td" contains text "Artist"
     And element ".uitest-level-dialog-content td .fa-plus" is visible
-    And I press the first ".uitest-level-dialog-content td .fa-plus" element
+    And I press ".uitest-level-dialog-content td .fa-plus:first" using jQuery
```

Comment:
in general we shouldn't be using jQuery, but I ran into a number of issues using the selenium clicks. On this line, Firefox couldn't click the `+` button to add a level presumably because the dialog is partially off the screen:
![Screen Shot 2020-10-26 at 5 46 52 PM](https://user-images.githubusercontent.com/8001765/97328497-5138ae80-1833-11eb-905c-fe9660658bef.png)

Reply:
Should we be concerned that the `+` is off screen or do we think this is just a test quirk?

Reply:
yes, probably... I can repro the `+` column being off screen when I shrink my window to 1024px wide.

Reply:
I've added this problem to the cplat internal bug bash doc.